packages/backend/migrations/20250121193606_testing.down.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM _sqlx_migrations WHERE version = '20250121193606';
```

Comment:
Why is this in a migration? This is very concerning, especially without any comment on why it's needed.

If you want to rewrite migration history it's not advisable to do it from inside a migration, it typically should be done manually. It can technically work sometimes, but really sucks when it doesn't.